CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # esrap changelog
 
+## 1.1.1
+
+- Actually add the changes that were supposed to go in 1.1.0
+
+## 1.1.0
+
+- Tweak output ([#5](https://github.com/Rich-Harris/esrap/pull/5))
+
```

Comment:
err what

should probably push this stuff to main separately before merging this PR

Reply:
huh. i think this is because i was dabbling with pierre instead of github, and something fell through the cracks